pontoon/sync/tests/formats/test_xliff.py

```diff
@@ -142,42 +142,6 @@ def test_save_basic(self):
             input_string, expected_string, source_string=input_string
         )
 
-    def test_save_preserve_target_language(self):
```

Comment:
Shouldn't we restore this test, and test for both iOS and non-iOS projects?

Reply:
This test will fail, because it expects `target-language` to not be changed if it exists. This is no longer the case, because we always set it (the `if` from https://github.com/mozilla/pontoon/pull/2876#pullrequestreview-1482109243 is gone).

We should add a test to check different behavior for iOS projects. I didn't include it, because I'd like to handle it differently in a followup - I'd like to introduce `Locale.ios_code` and read iOS locales from the data model rather than hardcode them. We do something similar for Google Translate and Microsoft Translator locales.